docs/webpack.config.js

```diff
@@ -33,5 +34,14 @@ module.exports = {
       }
     ]
   },
-  plugins: [new ExtractTextPlugin('[name].css')]
+  plugins: [
+    new webpack.LoaderOptionsPlugin({
```

Comment:
What is the utility to use [LoaderOptionsPlugin](https://webpack.js.org/plugins/loader-options-plugin/) if we can set an option in [the loader itself](https://github.com/shama/stylus-loader/issues/178#issuecomment-331740327)?

Reply:
Thanks, good catch, done !